backends/ebpf/CMakeLists.txt

```diff
@@ -221,6 +221,14 @@ else()
   set (SUPPORTS_KERNEL False)
 endif()
 
+# List of executable dependencies
+set(TEST_DEPENDENCY_PROGRAMS tcpdump)
+
+# List of library dependencies
+set(TEST_DEPENDENCY_LIBRARIES libpcap-dev gcc-multilib scapy)
```

Comment:
There are a couple issues here. `libpcap-dev` should just be `pcap` to be found. Scapy as a binary is not required. Instead we need the python library because that is what is used in the tests. `find_package` should work. 

For `gcc-multilib` I am actually not sure what is needed here. I do not think we even require it. You can remove it.

https://github.com/p4lang/p4c/blob/main/tools/ci-build.sh#L158 is what we need for the eBPF tests. We should have checks for all this. 


So the checks have to be a bit more granular.

Reply:
Thanks for the fixes. I believe this part is still missing.